theme/themes/pastanaga/extras/grid.less

```diff
@@ -0,0 +1,405 @@
+// Row block
+@blueArctic: #e2f1fd;
+@greySnow: #f3f5f7;
+@greySmoke: #e4e8ec;
+@grid-images-aspect-ratio: var(--grid-images-aspect-ratio, 16/9);
+@grid-images-object-position: var(--grid-images-object-position, top left);
+
+.block-editor-gridBlock {
+  position: relative;
+  // Removed because it broke the z-index of the blockChooser, making it fall behind
+  // z-index: 10;
+
+  .block {
+    height: 100%;
+
+    .image {
+      & > div {
+        height: 100%;
+      }
+
+      & > div > div {
+        height: 100%;
+      }
+    }
+  }
+}
+
+// Override handler TODO: do it better (only for grids)
```

Comment:
What's left to do here?

Reply:
It was a marker for when we use the Quanta Toolbar... nothing important, I completed it.